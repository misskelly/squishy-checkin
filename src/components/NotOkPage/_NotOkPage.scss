.not-ok-section {
  background: $light-text-color;
  font-family: $main-font;
}

.not-ok-heading {
  font-family: $heading-font;
  margin: 0 auto;
  @include font-size(7rem);
  font-weight: bold;
  display: block;
  line-height: 1;
  padding: 10rem;

}

.not-alone-text {
  text-transform: uppercase;
  font-weight: 300;
  text-shadow: $main-text-color 0 .5px 1px;
  line-height: 1;
  @include font-size(2rem);
}


.heart {
  margin-top: 2rem;
  display: block;
  @include font-size(5rem);
}
.arrow {
  transform: rotate(180deg);
  height: 30rem;
  filter: invert(0.4) sepia(1) saturate(1.4) hue-rotate(158.4deg) brightness(0.5);
  padding-top: 5rem;
}

.safety-wrapper {
  @include article-wrapper;
  width: 70%;
  margin: 0 auto 10rem auto;
  padding: 2rem;
  // margin-bottom: 10rem;
}

.safety-list {
  text-align: left;
  padding-bottom: 2rem;
  line-height: 2;
}

.crisis-btn {
  text-decoration: none;
  margin: 0 auto;
  @include button;
  text-transform: uppercase;
}

.crisis-q-heading {
  padding: 5rem 0 15rem 0;
  font-family: $heading-font;
  font-weight: bold;
  margin: 2rem;
  @include font-size(5rem);
}

.crisis-q-article {
  width: 50%;
  margin: 5rem auto;
}
.crisis-q {
  @include font-size(2rem);
  font-family: $heading-font;
  font-weight: 300;
  text-transform: uppercase;
}

.crisis-a {
  @include font-size(1.1rem);
}
.q-arrow {
  height: 15rem;
  transform: rotate(180deg);
  filter: invert(0.4) sepia(1) saturate(1.4) hue-rotate(158.4deg) brightness(0.5);
  margin: 20rem auto 3rem auto;
}

.underwater-img {
  width: 80%;
  max-width: 700px;
}

