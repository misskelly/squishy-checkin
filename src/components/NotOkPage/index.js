import React from 'react';
import { crisisQuestions } from '../../data';
import uniqid from 'uniqid';

export const NotOkPage = () => {
  const questions = crisisQuestions.map((q) => {
    const id = uniqid();
    return (
      <article
        className="crisis-q-article"
        key={id}
      >
        <h4 className="crisis-q">
          {q.question}
        </h4>
        <p className="crisis-a">
          {q.answer}
        </p>
      </article>

    );
  });
  return (
    <section className="not-ok-section">
      { questions }
    </section>
  );
};

export default NotOkPage;
