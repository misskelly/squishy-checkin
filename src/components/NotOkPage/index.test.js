import React from 'react';
import { shallow } from 'enzyme';
import NotOkPage from './index'

describe('NotOkPage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotOkPage />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});