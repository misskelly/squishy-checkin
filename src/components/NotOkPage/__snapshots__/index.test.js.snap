// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotOkPage should match the snapshot 1`] = `ShallowWrapper {}`;
